docs/source/app-dev/bindings-java/quickstart/template-root/daml/Tests/Trade.daml

```diff
@@ -12,12 +12,13 @@ import Iou
 import IouTrade
 
 trade_test = do
-  alice <- allocateParty "Alice"
-  bob <- allocateParty "Bob"
-  usBank <- allocateParty "USD_Bank"
-  eurBank <- allocateParty "EUR_Bank"
+  -- allocate parties
+  alice <- allocatePartyWithHint "Alice" (PartyIdHint "Alice")
```

Comment:
why do you need hints here?

Reply:
just so this matches the initialize script in `Main.daml`